### PR TITLE
Add feature one

### PR DESCRIPTION
**Pull request by agalit1**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -2,5 +2,6 @@ public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
         System.out.println("Hello git!");
+        System.out.println("add feature one");
```
Comment (chris-yooo) [inline src/Main.java:5]: joar is ok 4 me :)

**Comment by Marc120985** [comment]
Finde ich gut

**Review by chris-yooo — APPROVED** [review]

File: src/Main.java (hunk 1 above)
Comment (agalit1) [inline src/Main.java:5]: well done!

**Comment by Mykotine** [comment]
excellent!👌

**Pull request merged**